test/es6/es6IsConcatSpreadable.js

```diff
@@ -5,12 +5,12 @@
 
 WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
 
-function areEqual(a,b)
+function areEqual(a,b,msg)
 {
-    assert.areEqual(a.length,b.length);
+    assert.areEqual(a.length,b.length,msg);
     for(var i = 0;i < a.length; i++)
     {
-        assert.areEqual(a[i],b[i]);
+        assert.areEqual(a[i],b[i],msg);

```

Comment:
Added in the latest rev.

---
In reply to: [68806409](https://github.com/Microsoft/ChakraCore/pull/1198#discussion_r68806409) [](ancestors = 68806409)